src/lib/holocene/accordion.svelte

```diff
@@ -16,11 +17,11 @@
 >
   <div class="w-full">
     <div
-      class="accordion-open flex cursor-pointer flex-col"
+      class="accordion-open flex {!readOnly ? 'cursor-pointer' : ''} flex-col"
```

Comment:
Is this way of adding conditional classed preferred vs. doing something like `class:readOnly`?